src/jmh/java/pl/allegro/tech/opel/OpelEngineBanchmark.java

```diff
@@ -1,6 +1,6 @@
 package pl.allegro.tech.opel;
 
-import org.openjdk.jmh.annotations.Benchmark;
+//import org.openjdk.jmh.annotations.Benchmark;
```

Comment:
nie zakładamy, zależność na to wynika z jmh-gradle-plugin. Jeśli jest jakiś problem to z idea ze traktuje sourceSet jako main ale nie wczytuje poprawnie zaleznosci. Z poziomu gradle wszystko jest ok. Dlatego impotr ten musi zostać. Jeśli komuś przeszkadza jak działa idea, to może albo ręcznie excludowac katalog jmh lub w gradle dodać idea plugi i to ustawic.